src/code/z_bg_collect.c

```diff
@@ -1,9 +1,4 @@
-#include <ultra64.h>
-#include <global.h>
-
-// This file comes after z_bgcheck.c and contains functions to manage actors attached to Bg Actors.
-// Perhaps it is called something like z_bgcheck_attach.c
-// The functions will use the BgCheck2 prefix for now.
```

Comment:
Should the function names change now the file has a name I assume is more definitive?

Reply:
no